### ISSUE TITLE
{error} Initial Registration failed [PLMN_NOT_ALLOWED]

### ISSUE THREAD

**Issue by EdoanRadiance** [aligungr/UERANSIM#566]
I have a configuration of open5gs and UERANSIM here, with amf/smf installed on vm1, upf on vm2, gnb on vm3, and ue on vm4.
I have mcc: 001 and mnc: 01 on all configuration files relevant. I am new to using github, so if more information is needed please ask me.
I have run through this configuration once before without problems, so I do not know what is wrong now.
![gnb](https://user-images.githubusercontent.com/113628572/193868504-73988dd1-2eda-4e37-8d5b-e74cc84edb66.png)

![ue](https://user-images.githubusercontent.com/113628572/193868682-0d918dca-464b-4d95-981f-647ef3629c39.png)
![amf](https://user-images.githubusercontent.com/113628572/193869920-7c8d9e96-b401-496e-b046-f818bd0715f3.png)
![smf](https://user-images.githubusercontent.com/113628572/193869932-6d9437ca-a08c-47df-9312-fa4fd02cb232.png)

**Comment by Neruzzz** [comment]
I have teh same error. Have you solved it??

**Comment by EdoanRadiance** [comment]
Hello, yes I have. It was very silly of me. I had everything set up correctly, but had my subscriber information logged in the user plane vm instead of the control plane vm. Hope this helps.

**Comment by Neruzzz** [comment]
Yes, i solved it. It whas a mixture of not having the user created on the control plane and then not configuring both planes. All on the WebUI. Thanks.

**Comment by Poonamsurwase** [comment]
Hi EdoanRadiance

 I am facing same issue , could u please mention the detail configuration of open5gc and UERANSIM with amf/smf 
The command which u have used 
And the solution for it 

Thanks in advance

**Comment by EdoanRadiance** [comment]
It might be easier if you were to share your current configuration of your
files, as ours will most likely not be the same. Provide screenshots if you
will.
